interface/main/calendar/find_group_popup.php

```diff
@@ -0,0 +1,306 @@
+<?php
+/* Copyright (C) 2005-2007 Rod Roark <rod@sunsetsystems.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ */
```

Comment:
The js functions are different and the html is different. I thought it's more orderly to put in seperate file. Would you like me to merge them? Also - I think the name 'find_patient_popup' isn't fitting for groups so even if we don't rewrite the code I think we should keep this seperate file and require the find_patient_popup file. What do you say?